VariationalBayes/ExponentialFamilies.py

```diff
@@ -116,6 +117,37 @@ def get_e_log_dirichlet(alpha):
     digamma_sum = sp.special.digamma(np.sum(alpha, 0, keepdims=True))
     return sp.special.digamma(alpha) - digamma_sum
 
+gh_loc, gh_weights = hermgauss(4)
+def get_e_logitnormal(lognorm_means, lognorm_infos, \
+            loc = gh_loc, weights = gh_weights):
+    # get the expectation of a logit normal distribution
+    e_sticks = np.array([np.dot(1/np.sqrt(np.pi) * weights,  \
+                sp.special.expit(np.sqrt(2) * loc * \
+                1/np.sqrt(lognorm_infos[k]) +\
+                lognorm_means[k])) \
+                for k in range(np.size(lognorm_means))])
```

Comment:
Here and below, please do not use for loops -- they are very expensive, especially for autodiff.  Instead, make an array of the transformed gh locations and evaluate the function at those locations.